ggircs-app/app/next-env.d.ts

```diff
@@ -22,9 +22,9 @@ export type PageComponent = NextComponentType<
   PageComponentProps
 > &
   ComponentClass<PageComponentProps> & {
-    static query: GraphQLTaggedNode;
-    static isAccessProtected: boolean;
-    static allowedGroups: string[];
+    query: GraphQLTaggedNode;
+    isAccessProtected: boolean;
+    allowedGroups: string[];
```

Comment:
Why this change?

Reply:
it looks like pre-commit did this? I changed it back to include the static keywords and pre-commit auto-changed it again :upside_down_face:

Reply:
Should these stay `static` or is pre-commit right? To keep them as `static` I can change them back and set pre-commit to ignore this file.

Reply:
try updating prettier and see if that changes anything? Otherwise if that's not causing any issue at compile time I think we can let pre-commit do its thing

Reply:
updating prettier had no effect, leaving this as is.